### DISCUSSION TITLE
WebGUI img2img will break internet connection randomly

### DISCUSSION THREAD

**Discussion by nonetrix** [hlky/stable-diffusion#207] [General]
For some reason when I click generate on images in img2img tab it will randomly disconnect the internet(WiFi) it will still show as connected but pages will not load even localhost is broken 

This seems very odd and I have not been able to reproduce it consistently seems random but very common and doesn't happen doing anything else I cannot get any useful error logs

OS:
```
██████████████████  ████████   user@user-b450aorusprowifi 
██████████████████  ████████   -------------------------- 
██████████████████  ████████   OS: Manjaro Linux x86_64 
██████████████████  ████████   Host: B450 AORUS PRO WIFI 
████████            ████████   Kernel: 5.15.60-1-MANJARO 
████████  ████████  ████████   Uptime: 5 mins 
████████  ████████  ████████   Packages: 891 (pacman), 36 (flatpak) 
████████  ████████  ████████   Shell: zsh 5.9 
████████  ████████  ████████   Resolution: 1920x1080 
████████  ████████  ████████   DE: GNOME 42.4 
████████  ████████  ████████   WM: Mutter 
████████  ████████  ████████   WM Theme: Adwaita 
████████  ████████  ████████   Theme: Adw-dark [GTK2/3] 
████████  ████████  ████████   Icons: Papirus-Dark [GTK2/3] 
                               Terminal: gnome-terminal 
                               CPU: AMD Ryzen 7 3700X (16) @ 3.600GHz 
                               GPU: NVIDIA GeForce GTX 1080 
                               Memory: 8022MiB / 32040MiB 
```

>I had this issue getting my docker running. The problem had to do with folder permissions when installing conda, so it wasn't doing it at all.

>CONDA_DIR /opt/conda
>wget -O ~/miniconda.sh -q --show-progress --progress=bar:force https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh && \
    >/bin/bash ~/miniconda.sh -u -b -p $CONDA_DIR && \
    >rm ~/miniconda.sh
>PATH=$CONDA_DIR/bin:$PATH
>conda update -n base -c defaults conda

>This fixed it for me. Although I wasn't going to recommend edits because I think it's due to an Unraid quirk

note I also used the fixes listed here as it didn't work the first try https://github.com/hlky/stable-diffusion/issues/129

**Comment by hlky** [comment]
Moved to discussion.
